l10n_eu_product_adr/wizards/DG_product_handler.py

```diff
@@ -0,0 +1,131 @@
+# Copyright 2020 Iryna Vyshnevska (Camptocamp)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+
+from itertools import groupby
+
+from odoo import fields, models, api
+
+
+class DGProductCounter(models.TransientModel):
+    _name = "dangerous.goods.handler"
+    _description = "Wizard to count and prepare data for dangerous goods report"
+
+    picking_ids = fields.Many2many("stock.picking", string="Pickings")
+
+    @api.multi
+    def prepare_DG_data(self):
+        self.ensure_one()
```

Comment:
@Kiplangatdan Actually Travis complains now that there is code before the docstring, it needs to be after:

l10n_eu_product_adr/wizards/DG_product_handler.py:39: [W0105(pointless-string-statement), DGProductCounter.prepare_DG_data] String statement has no effect